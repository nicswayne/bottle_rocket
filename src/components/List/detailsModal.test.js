import React from 'react';
import { shallow, render } from 'enzyme';

import DetailsModal from './detailsModal';

const details = {
    name: 'name',
    category: 'category',
    location: {
        address: 'address',
        city: 'city',
        lat: 1,
        lng: 2,
        postalCode: 12345,
        state: 'ST',
    },
    contact: {
        formattedPhone: 'phone',
        twitter: 'twitter',
    },
};

it('DetailsModal component should render without crashing.', () => {
    shallow(<DetailsModal google={{}} details={details} />);
});

it('DetailsModal component should match snapshot.', () => {
    const wrapper = render(<DetailsModal google={{}} details={details} />);

    expect(wrapper).toMatchSnapshot();
});
