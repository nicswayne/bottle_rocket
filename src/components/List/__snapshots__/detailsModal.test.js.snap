// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailsModal component should match snapshot. 1`] = `
<div>
  <div
    class="loc-map"
  >
    <div
      class="map-content"
    >
      <div
        style="position:relative;width:100%;height:180px"
      >
        <div
          style="position:absolute;left:0;right:0;bottom:0;top:0;display:inherit"
        >
          Loading map...
        </div>
      </div>
    </div>
  </div>
  <div
    class="details-header"
  >
    <div
      class="name text"
    >
      name
      <div
        class="category"
      >
        category
      </div>
    </div>
  </div>
  <div>
    <div
      class="name pad-16"
    >
      address
      city, ST 12345
    </div>
    <div
      class="name pad-26"
    >
      phone
    </div>
    <div
      class="name pad-26"
    >
      @twitter
    </div>
  </div>
</div>
`;
